jerry-debugger/src/lib/debugger-client.ts

```diff
@@ -77,4 +77,10 @@ export class JerryDebuggerClient {
   onMessage(data: ArrayBuffer) {
     this.delegate.onMessage(new Uint8Array(data));
   }
+
+  send(data: ArrayBuffer) {
+    if (this.socket) {
+      this.socket.send(data);
+    }
```

Comment:
throw an error if it's not there? For now at least... probably better than silently dropping the data.

Reply:
OK... how about using ! here... it should still throw an exception if it ends up being undefined, right?

Reply:
Yes, it would throw an exception, AFAIK, `!` is just an override/hint to the TS compiler.